include/mbgl/style/layers/layer.hpp.ejs

```diff
@@ -44,6 +44,13 @@ public:
 <% } -%>
 
 <% } -%>
+    // Visibility
+    void setVisibility(VisibilityType) final;
```

Comment:
You want to write `override final` or `final override`? Neither of those appear elsewhere in the codebase.